packages/http-client-java/emitter/src/code-model-builder.ts

```diff
@@ -992,43 +995,147 @@ export class CodeModelBuilder {
   private processRouteForPaged(
     op: CodeModelOperation,
     responses: SdkHttpResponse[],
-    sdkMethod: SdkMethod<SdkHttpOperation>,
+    sdkMethod: SdkServiceMethod<SdkHttpOperation>,
   ) {
     if (sdkMethod.kind === "paging" || sdkMethod.kind === "lropaging") {
       for (const response of responses) {
         const bodyType = response.type;
         if (bodyType && bodyType.kind === "model") {
-          let itemSerializedName: string | undefined = undefined;
-          let nextLinkSerializedName: string | undefined = undefined;
-
-          const itemSegments = sdkMethod.response.resultSegments;
-          const nextLinkSegments = sdkMethod.pagingMetadata.nextLinkSegments;
+          op.responses?.forEach((r) => {
+            if (r instanceof SchemaResponse) {
+              this.trackSchemaUsage(r.schema, { usage: [SchemaContext.Paged] });
```

Comment:
Probably it's because I don't have the full context – but I wonder why we track schema usage for ALL "op" responses for EACH response in "method"

Reply:
Good catch.

There is a "break" a few hundred lines later, so only the 1st response is processed -- but that's too cryptic.

For normal op, one can write `op get(): Model1 | Model2` for more than 1 responses. For my try, for pageable op there would be a lint error (as there would be 2 different model to page from).

Therefore I just simplified the logic to process only 1 `sdkMethod.responses`.